src/posts/us-exit-documents/index.md

```diff
@@ -0,0 +1,307 @@
+If you're interested in hosted Nextcloud I urge you to check out the above and talk to their sales departments about price.
+
+## Jottacloud
+
+[Jottacloud](https://jottacloud.com/en/) is a Norwegian company focused on cloud storage.
+They offer various plans for both persons and business and have a strong focus on backing up your devices.
+
+For document editing, they use Microsoft Office Online, which sends your documents to Microsoft's servers while editing and then syncs the changes back to Jottacloud.
+This makes them a no-go for factor10, unfortunately.
+
+Syncing on Linux is done via their own CLI application and as far as I can tell, it's immediate and not scheduled.
+
+They offer various plans for both personal and business use.
+The one that would suit me is SEK 149 (about EUR 13.5) for 5 TiB of storage with up to five users.
+What that offer doesn't say however is that those 5 TiB is per user, so each user gets 1 TiB of storage and you can't use more than that on each account.
+
+## pCloud
+
+[pCloud](https://pcloud.com) is a Swiss cloud storage provider that offers lifetime storage for a one-time fee.
+They have plans for personal, family and business use and the family plan includes up to 5 users and comes in a 2 TiB and 10 TiB flavour for EUR 595 and EUR 1499 respectively.
+
+They don't have any collaboration features as far as I can tell and the 2/10 TiB plans are either too small or excessively large.
+Paying EUR 1499 is also a lot of money even if it's a one-time payment for a lifetime of storage &mdash; you're also betting that pCloud will outlive you.
+
+# The decision
+
+## Me
+
+I've gone with Infomaniak kDrive Team.
+The 3 TiB storage fits perfectly and with 6 users I can also give my parents their own accounts so they can store their stuff there as well.
+EUR 11.11 per month is cheaper than what I pay for OneDrive, which is a nice bonus.
+
+kDrive has no photo management so we'll lose "On this day", I'll have to replace that with something else.
+Probably something running on the home server.
+
+## factor10
+
+factor10 is also going with Infomaniak, but with kSuite Pro and not kDrive.
+The price is lower than for kDrive Pro and in the end will give us more storage (not that we need it).
+Going with kSuite also allows us to check out their other offerings and evaluate whether they can replace any other services
+(spoiler: we've already switched from Slack to kChat and from Google Meet to kMeet).
+
+# Final notes
+
+Personally, kDrive was the best offer both feature-wise and price-wise.
+I actually signed up for JottaCloud and I like their service in general &mdash; from what I saw, they have very good photo management.
+Their sharing is too limited for me though, and the fact that I won't be able to use 2 TiB of storage on one account is a deal-breaker.
+
+factor10 went with Infomaniak because of several reasons. Chief among them though, is that they give off a good impression.
```

Comment:
I think it would be good with a comment about we don't see this as a dramatic decision. If/when Infomaniak isn't considered right/good enough in the future, we move again...?